admin/client/src/components/Accordion/AccordionGroup.js

```diff
@@ -15,12 +15,10 @@ class AccordionGroup extends SilverStripeComponent {
       'aria-labelledby': headerID,
     };
     return (
-      <div className="accordion-group">
-        <h6 className="accordion-group__title" role="tab" id={headerID}>
-          <a data-toggle="collapse" href={href} aria-expanded="true" aria-controls={listID}>
-            {this.props.title}
-          </a>
-        </h6>
+      <div className="accordion">
```

Comment:
This seems wrong, the component is called `AccordionGroup`, but the class is `accordion`. We still have an `Accordion` component as the parent of this one, right?